src/ui/components/shared/Anchor.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { AnchorHTMLAttributes, ReactNode } from "react";
+
+export default function Anchor({
```

Comment:
I dunno about you, but `Anchor` doesn't mean much to me in a vacuum. I know `Link` is easily confused with `Next/Link`, but what if we called this like `ExternalLink`, since that's really what it's doing, making it safe to link to external resource?

Reply:
Switched to ExternalLink